python/gstaichi/lang/kernel_impl.py

```diff
@@ -1256,6 +1293,7 @@ def wrapped_func(*args, **kwargs):
     wrapped._is_classkernel = is_classkernel
     wrapped._primal = primal
     wrapped._adjoint = adjoint
+    primal.gstaichi_callable = wrapped
```

Comment:
The standard dunder attribute name for wrapped function is `__wrapped__`

Reply:
You mena, rename `gstaichi_callable` to `__wrapped`? What unintended consequences are possible if I do that?